interfaces/acados_matlab_octave/ocp_solve.c

```diff
@@ -40,10 +40,19 @@
 // mex
 #include "mex.h"
 
-
+#if defined(_WIN32) || defined(__WIN32__) || defined(WIN32) || defined(_WIN64)
```

Comment:
While all of these changes are necessary to show the verbose output on Windows Matlab, they are annoying when you don't want verbose output, especially when you are running a closed-loop controller: it will open and close a terminal window for each call to `ocp_solve`, thus showing a flickering terminal window during the entire closed loop simulation (which also slows down the computation). If you don't feel like implementing to activate these lines only when the print_level > 0, I'd leave them commented out by default

Reply:
I agree! I committed it in uncommented form because I thought it would pipeline it to the "normal" Matlab terminal.